packages/fe-fpm-writer/test/unit/section.test.ts

```diff
@@ -136,7 +136,7 @@ describe('CustomSection', () => {
             expect(fs.read(fragmentPath)).toMatchSnapshot();
         });
 
-        const testVersions = [1.9, 1.85, 1.84,  1.86, 1.90];
+        const testVersions = [1.9, 1.85, 1.84, 1.86, 1.9];
```

Comment:
Why do we have `1.9` twice?

Reply:
Why is this change in this PR? This is a regression currently in #269 which @stefanschreck is going to fix before merging. It should not be here ...

Reply:
@tobiasqueck: It looks like it was updated by prettier in the git hook. I'll undo this change.